ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -51,20 +51,39 @@ $services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&
   'leader': {"type":"text","size":10,"html":true},
   'place': {"type":"text","size":10},
   'startDate': {"type":"text","size":10},
+  'endDate': {"type":"hidden"},
   'durationDisplay': {"type":"text","filterable":true,"sortable":false,"html":true},
   'status': {"type":"list","size":10},
   '_actions': {"sortable":false,"filterable":false,"html":true,"actions":["delete", "edit"]}
 })
-#set ($options = {
-  'className': 'Meeting.Code.MeetingClass',
-  'translationPrefix': 'meeting.livetable.',
-  'tagCloud': true,
-  'rowCount': 15,
-  'maxPages': 10,
-  'selectedColumn': 'startDate',
-  'defaultOrder': 'desc'
-})
-#set ($columns = ['doc.title', 'leader', 'place', 'startDate', 'durationDisplay', 'status', '_actions'])
+#set ($currentDate = $xwiki.jodatime.dateTime.millis)
+#set ($endDate = $xwiki.jodatime.dateTime.plusYears(1).millis)
+#set ($parameter= $request.getParameter("meeting"))
+#if($parameter=="all")
```

Comment:
Wrong indentation: `#if ($parameter == "all")`

Reply:
Single quotes should also be used `'all'`, same as for `'meeting'`